doc/read-the-docs-site/quick-start.rst

```diff
@@ -0,0 +1,127 @@
+.. _quick_start:
+
+Quick Start
+=================================
+
```

Comment:
The title "Quick Start" is a little bit vague. Maybe it could be more specific, like "Setting up the dev environment" or "Setting up the dev environment for writing an on-chain validator script". When I looked at the Tutorials section, I wanted to understand what the relationship is between this Quick Start and one of the tutorials that is currently there called "Writing basic validator scripts." I agree with @michaelpj that it would likely make sense to group this with the other tutorials.

Reply:
I'm going to disagree on this. I think it should be either "Quick Start" or "Getting Started". These two phrases are very standard, eye-catching and attractive. Whenever I want to learn how to use something (say X), the first think I google is always "X quick start" or "X getting started".